extension_helpers/_openmp_helpers.py

```diff
@@ -127,15 +139,55 @@ def get_openmp_flags():
 
         include_path = _get_flag_value_from_var('-I', 'CFLAGS')
         if include_path:
-            compile_flags.append('-I' + include_path)
+            for _ in include_path:
+                compile_flags.append('-I' + _)
+
+        include_path = _get_flag_value_from_var('-I', 'CXXFLAGS')
+        if include_path:
+            for _ in include_path:
+                compile_flags.append('-I' + _)
 
         lib_path = _get_flag_value_from_var('-L', 'LDFLAGS')
         if lib_path:
-            link_flags.append('-L' + lib_path)
-            link_flags.append('-Wl,-rpath,' + lib_path)
-
-        compile_flags.append('-fopenmp')
-        link_flags.append('-fopenmp')
+            for _ in include_path:
```

Comment:
Should this be `lib_path`?  Right now I'm not picking up any `LDFLAGS` from the environment.

Reply:
Yes, it is. Thanks. Fixed by e93d695